CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Implements the `Attributes` trait for all R vector wrapper structs (e.g. `Integers` , `Doubles`, `Strings`, etc.), allowing for easy access and setting of the attributes of an R object [[#745]](https://github.com/extendr/extendr/pull/745). This comes with breaking changes. See below.
 - feature `non-api` that gives access to non-API items; Requires compile-time generation of bindings
 [[#754]](https://github.com/extendr/extendr/pull/754)
+- `TryFrom<&Robj>` for `StrIter`, `HashMap<K, Robj>` for `K = String` and `K = &str` [[#759]](https://github.com/extendr/extendr/pull/759)
```

Comment:
We need to make a statement here that this is a breaking PR `from_robj()` no longer exists nor does `use_try_from = true`. 

replace instances of `from_robj()` with `try_from()`